manim/renderer/opengl_renderer_window.py

```diff
@@ -0,0 +1,29 @@
+from .. import config
+import moderngl_window as mglw
+from moderngl_window.context.pyglet.window import Window as PygletWindow
+from moderngl_window.timers.clock import Timer
+
+
+class Window(PygletWindow):
+    fullscreen = False
+    resizable = True
+    gl_version = (3, 3)
+    vsync = True
+    cursor = True
+
+    def __init__(self, size=None, **kwargs):
+        if size is None:
+            size = (config["pixel_width"], config["pixel_height"])
+        super().__init__(size=size)
+
+        self.pressed_keys = set()
+
+        self.title = "Title goes here"
```

Comment:
better title than this?
Maybe `ManimCommunity OpenGL renderer (<version-goes-here>)`?